internal/scanner/scanner_test.go

```diff
@@ -375,7 +404,9 @@ func TestScanClusterWideResources(t *testing.T) {
 
 	scanner, err := NewScanner(policiesClient, k8sClient, policyReportStore, false, false, true, "")
 	require.NoError(t, err)
-	err = scanner.ScanClusterWideResources(context.Background())
+
```

Comment:
Could we add an old report here too, so we are sure that we are testing the deletion of `ClusterPolicyReports` too?

Reply:
Ops, thanks! I forgot to update this test.